hoodie-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/HoodieInputFormat.java

```diff
@@ -206,7 +206,8 @@ public Configuration getConf() {
    * Read the table metadata from a data path. This assumes certain hierarchy of files which should
    * be changed once a better way is figured out to pass in the hoodie meta directory
    */
-  protected static HoodieTableMetaClient getTableMetaClient(FileSystem fs, Path dataPath) {
+  protected static HoodieTableMetaClient getTableMetaClient(FileSystem fs, Path dataPath)
```

Comment:
I'm guessing this method is not called anywhere else so the extra throws IOException doesn't matter ?

Reply:
Checked.. Its called by listStatus or getSplits, both throw IOException. So should be ok.